dm/ctl/master/migrate_relay.go

```diff
@@ -59,9 +60,8 @@ func migrateRelayFunc(cmd *cobra.Command, _ []string) {
 		BinlogPos:  uint32(binlogPos),
 		Worker:     worker,
 	})
-
 	if err != nil {
-		log.Infof("can not migrate relay config:\n%v", errors.ErrorStack(err))
+		log.L().Error("can not migrate relay", zap.Error(err))
```

Comment:
I think `common.PrintLines` is better.

Reply:
I don't know why dmctl don't unify these logs  :confused:  , I suggest you can unify them in dmctl later in other PR

Reply:
OK